src/components/style.js

```diff
@@ -36,6 +36,14 @@ const Style = React.createClass({
 
       if (selector === 'mediaQueries') {
         accumulator += this._buildMediaQueryString(rules);
+      } else if (typeof styles[selector] !== 'object') {
+        // If the selector is actually a rule, and not a selector,
+        // prefix it with the scopeSelector
+        accumulator += cssRuleSetToString(
```

Comment:
If you have several properties (say color, background, border), won't this add a separate rule with the same selector for each one?